modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/FeaturedMediaFormatter.php

```diff
@@ -143,99 +141,38 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
   }
 
   /**
-   * Renders a single field item.
-   *
-   * @param \Drupal\Core\Field\FieldItemInterface $item
-   *   The individual field item.
-   * @param string $langcode
-   *   The language code.
-   *
-   * @return array
-   *   The ECL media-container parameters.
+   * {@inheritdoc}
    */
   protected function viewElement(FieldItemInterface $item, string $langcode): array {
-    $build = ['#theme' => 'oe_theme_helper_featured_media'];
-    $params = ['description' => $item->caption];
+    $pattern = [];
     $media = $item->entity;
-    $cacheability = CacheableMetadata::createFromRenderArray($build);
-
     if (!$media instanceof MediaInterface) {
-      return [];
+      return $pattern;
     }
-
+    $image_style = $this->getSetting('image_style');
+    $view_mode = 'oe_theme_main_content';
```

Comment:
We should add view_mode as a setting to use it with paragraph__oe_text_feature_media.